### PR TITLE
Ft admin add book 162739832 Adds the user POST endpoint

### PR DESCRIPTION
**Pull request by gkarumba**
#### What does this PR do?
It adds the user `POST` endpoint
#### Description of Task to be completed?
Create a books model and views file, implement adding a book feature
#### How should this be manually tested?
clone repo to the local machine, cd into the folder, run the following commands in the terminal in a virtualenv `pip install -r requirements.txt`then `flask run`. Open POSTMAN and test the endpoint
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#162739832
#### Questions:

### CODE REVIEW

File: app/api/books/v1/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from flask import Blueprint
+from flask_restful import Resource,Api
+#local imports
+from app.api.books.v1.views import AddBook
+
+bk1 = Blueprint('book_v1',__name__,url_prefix='/api/v1')
```
Comment (ipaullly) [inline app/api/books/v1/__init__.py]: I feel `bk1` is a bit ambiguous, why not something like `library_v1`

Comment (gkarumba) [reply app/api/books/v1/__init__.py]: let me change it

File: app/api/books/v1/models.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            
+            
+        
+        
+    def is_added(self,title):
+        """
+            Method for checking if a book is already in the library
+        """
+        for book in books_list:
+            if (title == book['title']):
```
Comment (ipaullly) [inline app/api/books/v1/models.py]: the code should work without the () that wraps the `if` conditional statement

File: app/api/books/v1/models.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        status = 'Available'
+        payload = {
+            'status' : status,
+            'title' : title,
+            'author' : author,
+            'category' : category,
+            'id' : len(books_list) + 1,
+            'added_on'  : datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+        } 
+        check_book = self.is_added(title)
```
Comment (ipaullly) [inline app/api/books/v1/models.py:31]: I don't think you need to check if the book title exists since you call the validation in the view already.

**Pull request merged**